common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -583,3 +583,20 @@ def index_dictionary(self):
         xblock_body["content_type"] = "Sequence"
 
         return xblock_body
+
+
+class SummaryFields(object):
```

Comment:
Also, in terms of DDD - and keeping our code in synch with the domain language - we may need to rename this once we determine what we're actually calling it.  It sounded like Product was leaning towards Section Highlights (and not summary) last.

Reply:
Sure. I'll just rename it right now to what they're leaning towards.